sharktank/tests/models/deepseek/test_sharded.py

```diff
@@ -8,21 +8,31 @@
 from copy import deepcopy
 
 import torch
-
-import unittest
-from copy import deepcopy
-
+import iree
+import pytest
 import torch
+import unittest
 
 from sharktank.models.deepseek.toy_deepseek import generate
 from sharktank.models.llm import PagedLlmModelV1
+from sharktank.types import Dataset
 from sharktank.types.sharding import shard_theta
 from sharktank.utils.evaluate import pad_tokens
+from sharktank.utils.export_artifacts import ExportArtifacts, IreeCompileException
 from sharktank.utils.load_llm import TorchGenerator
 from sharktank.utils.create_cache import *
+from sharktank.utils.iree import (
+    get_iree_devices,
+    load_iree_module,
+    TorchLikeIreeModule,
+    with_iree_device_context,
+)
+from sharktank.utils.testing import TempDirTestBase
+from sharktank.examples.sharding import shard_llm_dataset
 
 
-class DeepseekShardedTest(unittest.TestCase):
+@pytest.mark.usefixtures("get_iree_flags")
+class DeepseekShardedTest(TempDirTestBase):
```

Comment:
This is going to conflict with https://github.com/nod-ai/shark-ai/pull/1507/files#diff-0b3e0cc4c6ec4555ab5f4a5dcb68c931065378a7ef4e8374a0abdd2422ef316c

How do you want to proceed?